pkg/reconciler/instances/ory/action.go

```diff
@@ -47,13 +49,22 @@ type postDeleteAction struct {
 var (
 	jwksNamespacedName = types.NamespacedName{Name: "ory-oathkeeper-jwks-secret", Namespace: oryNamespace}
 	dbNamespacedName   = types.NamespacedName{Name: "ory-hydra-credentials", Namespace: oryNamespace}
+	rolloutHydra       = false
```

Comment:
How about getting rid of this var and read data from cluster dynamically? Such statefulness could harm us.

Reply:
Currently, we are limited by the db.Update method, we should open the new issue and work on the new Update logic.